<ui version="4.0" >
 <class>udg::QDicomDumpBase</class>
 <widget class="QDialog" name="udg::QDicomDumpBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Dicom dump</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QFrame" name="Frame" >
     <property name="minimumSize" >
      <size>
       <width>630</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="m_labelPatientName" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Patient Name :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLabel" name="m_labelPatientNameValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="m_labelPatientID" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Patient ID :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLabel" name="m_labelPatientIDValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="m_labelPatientSex" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Gender :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="m_labelPatientSexValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="m_labelPatientAge" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Age :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QLabel" name="m_labelPatientAgeValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="m_labelImageDate" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="acceptDrops" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>Image Date :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLabel" name="m_labelImageDateValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QLabel" name="m_labelImageTime" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Image Time :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QLabel" name="m_labelImageTimeValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="m_labelImageNumber" >
        <property name="font" >
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text" >
         <string>Image Number :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="QLabel" name="m_labelImageNumberValue" >
        <property name="text" >
         <string>-</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="udg::QDicomDumpMRWidget" native="1" name="m_qdicomDumpMRWidget" />
   </item>
   <item row="2" column="0" >
    <widget class="QDicomDumpCTSurveyWidget" native="1" name="m_qdicomDumpCTSurveyWidget" />
   </item>
   <item row="3" column="0" >
    <widget class="QDicomDumpCTWidget" native="1" name="m_qdicomDumpCTWidget" />
   </item>
   <item row="4" column="0" >
    <widget class="QDicomDumpCTHelixWidget" native="1" name="m_qdicomDumpCTHelixWidget" />
   </item>
   <item row="5" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pushButtonAccept" >
       <property name="text" >
        <string>Accept</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType" >
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>udg::QDicomDumpMRWidget</class>
   <extends>QWidget</extends>
   <header>qdicomdumpmrwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QDicomDumpCTSurveyWidget</class>
   <extends>QWidget</extends>
   <header>qdicomdumpctsurveywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QDicomDumpCTHelixWidget</class>
   <extends>QWidget</extends>
   <header>qdicomdumpcthelixwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QDicomDumpCTWidget</class>
   <extends>QWidget</extends>
   <header>qdicomdumpctwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
